C4SGSvc/src/main/java/org/c4sg/constant/ApplicationConstants.java

```diff
@@ -0,0 +1,11 @@
+package org.c4sg.constant;
+
+public class ApplicationConstants {
```

Comment:
Change for 2 **Enums** (follow the examples inside constants directory):
1) Called: Directory
    Variables: AVATAR_UPLOAD, RESUME_UPLOAD, etc
2) Called: Formats
    Variables: IMAGE, RESUME